src/obr/signac_wrapper/operations.py

```diff
@@ -263,6 +263,15 @@ def dispatch_pre_hooks(operation_name, job):
 def dispatch_post_hooks(operation_name, job):
     """just forwards to start_job_state and execute_pre_build"""
     execute_post_build(operation_name, job)
+    case = OpenFOAMCase(str(job.path) + "/case", job)
```

Comment:
I think this should be wrapped into a separate function and just called from here.